ml-algorithms/src/main/java/org/opensearch/ml/engine/tools/AgentTool.java

```diff
@@ -138,6 +139,11 @@ public String getDefaultType() {
         public String getDefaultVersion() {
             return null;
         }
+
+        @Override
+        public List<String> getRelatedModelIDKeyFields() {
```

Comment:
A better name could be `getAllModelKeys`

Reply:
Done